README.md

```diff
@@ -75,12 +83,36 @@ WebMock.disable_net_connect!(:net_http_connect_on_start => true, :allow_localhos
 ## Formatter modes
 
 The following modes are supported:
-
-| Name | Purpose |
-| --- | --- |
-| attach_to_launch | Do not create a new launch but add executing features/scenarios to an existing launch. Use launch_id or file_with_launch_id settings to configure that. If they are not present client will check rp_launch_id.tmp in `Dir.tmpdir`)
-| use_same_thread_for_reporting | Send reporting commands in the same main thread used for running tests. This mode is useful for debugging this Report Portal client. It changes default behavior to send commands in the separate thread. Default behavior is there not to slow test execution. |
-| skip_reporting_hierarchy | Do not create items for folders and feature files |
+<table><thead><tr><th>Name</th><th>Purpose</th></tr></thead>
```

Comment:
<img width="1007" alt="CapturFiles-20190808_084246" src="https://user-images.githubusercontent.com/40637521/62746372-1fb0a900-ba1d-11e9-9405-e97c32db9446.png">
I don't see any scroling

Reply:
https://github.github.com/gfm/#html-blocks - not sure what you are asking.

Reply:
As you can see on the screenshot you attached, not all text in column "Purpose" is visible. There is a horizontal scrolling (at least in Chrome).
https://github.com/reportportal/agent-ruby/blob/7be5353c31be361a2a73b93b83f8e4dffacd8c1b/README.md
![Screenshot from 2019-08-09 11-50-04](https://user-images.githubusercontent.com/1497867/62766803-f5520080-ba9b-11e9-888f-97db3da1fcc9.png)

Reply:
sure, but I don't see what is the issue here. If you know how to do multiple lines in a simple table block let me know.

Reply:
Previous formatting looked better IMO. Could you use it?

Reply:
That is subjective. Put everything in a single paragraph, and it will not look good in the browser IMO. Hense I had to change it to be more presentable.